Sources/Hyperspace/Request/AnyRequest.swift

```diff
@@ -9,6 +9,9 @@
 import Foundation
 import Result
 
+@available(*, deprecated, message: "The AnyNetworkRequest struct has been renamed AnyRequest.")
```

Comment:
removed can be used instead of message
`@available(*, deprecated: 2.0, renamed: "AnyRequest")`

Reply:
Fixed